### PR TITLE
Update md5 for new kcs.

### PR DESCRIPTION
**Pull request by JuneZhou19**
@weia5-emc @chenge3 @chenc52 @turtle-fly @XiaowenJiang @xiar

### CODE REVIEW

**Comment by JuneZhou19** [comment]
Update MD5 to avoid jenkins function CI test job failure.

**Comment by emcinfrasim** [comment]
```
******* Functional Test ******* 
TestName: test_set_disk_drive 
ErrorType: exceptions.IOError 
ErrorMessage: [Errno 2] No such file or directory: '/proc/-1/cmdline'
-------------------- >> begin captured stdout << ---------------------
[ 4245   ] test-socat starts to run
[ 4248   ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 4264   ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 4245   ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 4248   ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 4264   ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
TestName: test_copy_file_across_drives 
ErrorType: infrasim.InfraSimError 
ErrorMessage: 'Fail to download image kcs.img' 
TestName: test_file_existance_after_node_restart 
ErrorType: infrasim.InfraSimError 
ErrorMessage: 'Fail to download image kcs.img' 
TestName: test_controller_with_drive7 
ErrorType: exceptions.IOError 
ErrorMessage: [Errno 2] No such file or directory: '/proc/-1/cmdline'
-------------------- >> begin captured stdout << ---------------------
[ 8676   ] test-socat starts to run
[ 8679   ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 8696   ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sda.img
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sdb.img
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sdc.img
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sdd.img
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sde.img
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sdf.img
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sdg.img
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sdh.img
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sdi.img
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sdj.img
1Model: INFO: [        ] test-racadm is stopped
1Model: INFO: [        ] test-node is stopped
1Model: INFO: [        ] test-bmc is stopped
1Model: INFO: [        ] test-socat is stopped
1Model: INFO: [Node] Node test runtime workspcace is destroyed
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 8676   ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 8679   ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 8696   ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
TestName: test_controller_with_drive7_drive2_drive3 
ErrorType: exceptions.IOError 
ErrorMessage: [Errno 2] No such file or directory: '/proc/-1/cmdline'
-------------------- >> begin captured stdout << ---------------------
[ 8710   ] test-socat starts to run
[ 8713   ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 8729   ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 8710   ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 8713   ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 8729   ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
TestName: test_controller_with_drive6 
ErrorType: exceptions.IOError 
ErrorMessage: [Errno 2] No such file or directory: '/proc/-1/cmdline'
-------------------- >> begin captured stdout << ---------------------
[ 8747   ] test-socat starts to run
[ 8750   ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 8766   ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 8747   ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 8750   ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 8766   ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
TestName: test_controller_with_drive2 
ErrorType: exceptions.IOError 
ErrorMessage: [Errno 2] No such file or directory: '/proc/-1/cmdline'
-------------------- >> begin captured stdout << ---------------------
[ 8784   ] test-socat starts to run
[ 8787   ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 8803   ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 8784   ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 8787   ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 8803   ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
TestName: test_controller_with_drive6 
ErrorType: exceptions.IOError 
ErrorMessage: [Errno 2] No such file or directory: '/proc/-1/cmdline'
-------------------- >> begin captured stdout << ---------------------
[ 8946   ] test-socat starts to run
[ 8949   ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 8965   ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 8946   ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 8949   ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 8965   ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
TestName: test_controller_with_drive2 
ErrorType: exceptions.IOError 
ErrorMessage: [Errno 2] No such file or directory: '/proc/-1/cmdline'
-------------------- >> begin captured stdout << ---------------------
[ 8983   ] test-socat starts to run
[ 8986   ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 9002   ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 8983   ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 8986   ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 9002   ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
TestName: test_controller_with_drive6 
ErrorType: exceptions.IOError 
ErrorMessage: [Errno 2] No such file or directory: '/proc/-1/cmdline'
-------------------- >> begin captured stdout << ---------------------
[ 9020   ] test-socat starts to run
[ 9023   ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 9039   ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 9020   ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 9023   ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 9039   ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
TestName: test_controller_with_drive2 
ErrorType: exceptions.IOError 
ErrorMessage: [Errno 2] No such file or directory: '/proc/-1/cmdline'
-------------------- >> begin captured stdout << ---------------------
[ 9057   ] test-socat starts to run
[ 9060   ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 9077   ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 9057   ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 9060   ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 9077   ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
```

**Comment by emcinfrasim** [comment]
```
******* Functional Test ******* 
TestName: test_set_disk_drive 
ErrorType: exceptions.IOError 
ErrorMessage: [Errno 2] No such file or directory: '/proc/-1/cmdline'
-------------------- >> begin captured stdout << ---------------------
[ 12760  ] test-socat starts to run
[ 12763  ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 12779  ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 12760  ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 12763  ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 12779  ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
TestName: test_copy_file_across_drives 
ErrorType: socket.error 
ErrorMessage: [Errno 111] Connection refused
-------------------- >> begin captured stdout << ---------------------
[ 13262  ] test-socat starts to run
[ 13265  ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 13281  ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 13262  ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 13265  ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 13281  ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
TestName: test_file_existance_after_node_restart 
ErrorType: socket.error 
ErrorMessage: [Errno 111] Connection refused
-------------------- >> begin captured stdout << ---------------------
[ 13262  ] test-socat is already running
[ 13265  ] test-bmc is already running
[ ERROR ] test-node fail to start
[ 13288  ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 13262  ] test-socat is already running
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 13265  ] test-bmc is already running
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 13288  ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
TestName: test_chassis_power_cycle 
ErrorType: exceptions.AssertionError 
ErrorMessage: 
-------------------- >> begin captured stdout << ---------------------
[ 13262  ] test-socat is already running
[ 13265  ] test-bmc is already running
[ ERROR ] test-node fail to start
[ 13294  ] test-racadm starts to run
'pidof qemu-system-x86_64:'

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/disk00.img
1Model: INFO: test-socat's command line: socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 13262  ] test-socat is already running
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 13265  ] test-bmc is already running
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 13294  ] test-racadm starts to run
0Cmd: ERROR: pidof qemu-system-x86_64:
--------------------- >> end captured logging << --------------------- 
TestName: test_controller_with_drive7 
ErrorType: exceptions.IOError 
ErrorMessage: [Errno 2] No such file or directory: '/proc/-1/cmdline'
-------------------- >> begin captured stdout << ---------------------
[ 17121  ] test-socat starts to run
[ 17124  ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 17140  ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sda.img
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sdb.img
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sdc.img
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sdd.img
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sde.img
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sdf.img
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sdg.img
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sdh.img
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sdi.img
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/sdj.img
1Model: INFO: [        ] test-racadm is stopped
1Model: INFO: [        ] test-node is stopped
1Model: INFO: [        ] test-bmc is stopped
1Model: INFO: [        ] test-socat is stopped
1Model: INFO: [Node] Node test runtime workspcace is destroyed
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 17121  ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 17124  ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 17140  ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
TestName: test_controller_with_drive7_drive2_drive3 
ErrorType: exceptions.IOError 
ErrorMessage: [Errno 2] No such file or directory: '/proc/-1/cmdline'
-------------------- >> begin captured stdout << ---------------------
[ 17154  ] test-socat starts to run
[ 17157  ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 17173  ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 17154  ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 17157  ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 17173  ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
TestName: test_controller_with_drive6 
ErrorType: exceptions.IOError 
ErrorMessage: [Errno 2] No such file or directory: '/proc/-1/cmdline'
-------------------- >> begin captured stdout << ---------------------
[ 17191  ] test-socat starts to run
[ 17194  ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 17210  ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 17191  ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 17194  ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 17210  ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
TestName: test_controller_with_drive2 
ErrorType: exceptions.IOError 
ErrorMessage: [Errno 2] No such file or directory: '/proc/-1/cmdline'
-------------------- >> begin captured stdout << ---------------------
[ 17228  ] test-socat starts to run
[ 17231  ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 17247  ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 17228  ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 17231  ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 17247  ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
TestName: test_controller_with_drive6 
ErrorType: exceptions.IOError 
ErrorMessage: [Errno 2] No such file or directory: '/proc/-1/cmdline'
-------------------- >> begin captured stdout << ---------------------
[ 17390  ] test-socat starts to run
[ 17393  ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 17409  ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 17390  ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 17393  ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 17409  ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
TestName: test_controller_with_drive2 
ErrorType: exceptions.IOError 
ErrorMessage: [Errno 2] No such file or directory: '/proc/-1/cmdline'
-------------------- >> begin captured stdout << ---------------------
[ 17427  ] test-socat starts to run
[ 17430  ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 17446  ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 17427  ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 17430  ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 17446  ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
TestName: test_controller_with_drive6 
ErrorType: exceptions.IOError 
ErrorMessage: [Errno 2] No such file or directory: '/proc/-1/cmdline'
-------------------- >> begin captured stdout << ---------------------
[ 17464  ] test-socat starts to run
[ 17467  ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 17483  ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 17464  ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 17467  ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 17483  ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
TestName: test_controller_with_drive2 
ErrorType: exceptions.IOError 
ErrorMessage: [Errno 2] No such file or directory: '/proc/-1/cmdline'
-------------------- >> begin captured stdout << ---------------------
[ 17502  ] test-socat starts to run
[ 17505  ] test-bmc starts to run
[ ERROR ] test-node fail to start
[ 17521  ] test-racadm starts to run

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 17502  ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 17505  ] test-bmc starts to run
1Model: ERROR: [ ERROR ] test-node fail to start
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 17521  ] test-racadm starts to run
--------------------- >> end captured logging << --------------------- 
```

File: test/functional/test_host_io.py [hunk 1]
```diff
@@ -111,7 +111,7 @@ def prepare_ssh(self):
 
     def setUp(cls):
         DOWNLOAD_URL = 'https://github.com/InfraSIM/test/raw/master/image/kcs.img'
-        MD5_KCS_IMG = 'cfdf7d855d2f69c67c6e16cc9b53f0da'
+        MD5_KCS_IMG = '986e5e63e8231a307babfbe9c81ca210'
```
Comment (turtle-fly) [inline test/functional/test_host_io.py]: @JuneZhou19 it would be necessary to update the image to above `DOWNLOAD_URL` before this PR

**Comment by XiaowenJiang** [comment]
warning: host doesn't support requested feature: CPUID.07H:EBX.hle [bit 4]
warning: host doesn't support requested feature: CPUID.07H:EBX.erms [bit 9]
warning: host doesn't support requested feature: CPUID.07H:EBX.rtm [bit 11]
warning: host doesn't support requested feature: CPUID.07H:EBX.hle [bit 4]
warning: host doesn't support requested feature: CPUID.07H:EBX.erms [bit 9]
warning: host doesn't support requested feature: CPUID.07H:EBX.rtm [bit 11]
qemu-system-x86_64: Unable to connect character device serial0: Failed to connect socket: No such file or directory

**Comment by XiaowenJiang** [comment]
retest this please

**Comment by XiaowenJiang** [comment]
retest this please.
Check if the "serial device not found" issue will pop again.

File: test/functional/test_host_io.py (hunk 1 above)
Comment (JuneZhou19) [inline test/functional/test_host_io.py]: New image is already pushed into github~

**Pull request merged**